### ISSUE TITLE
The results of `FzfLua files` overflow on Ubuntu 20.04 with Fzf version `0.29.0` and `0.24.3`

### ISSUE THREAD

**Issue by nyngwang** [ibhagwan/fzf-lua#287]
## Fzf Version

```
✔ parallels ~ % fzf --version
0.29.0 (dc975e8)
```

## The problem:

As shown in the below demo. I remember that I have seen this bug before, and did search for the issue while I didn't find it.

The command I run is

```
lua require("fzf-lua").files()
```

https://user-images.githubusercontent.com/24765272/147813592-947aa98f-bbb7-4d75-a48b-2b2d3c5937d2.mov

**Comment by nyngwang** [comment]
OK from the last second, i.e. 0:08, it seems like it's because there is a path containing white spaces:

```
.../Parallels Shared Folders/Home/...
```

**Comment by ibhagwan** [comment]
>OK from the last second, i.e. 0:08, it seems like it's because there is a path containing white spaces

Just tested and I don't think this is related to spaces, you're running it on your home directory and it seems that `fd` is failing to access certain MacOS special folders and what you're seeing here is the error message returned by `fd`.

I really can't tell why `fd` fails but I can try to supress the error message as a workaround, regardless, it should not overflow into the `fzf` search field, we have to see what the maintainer of `fzf` says about this issue https://github.com/junegunn/fzf/issues/2697, which will potentially also solve #275.

**Comment by ibhagwan** [comment]
I think I know what causes this, as I suspected when certain folders fail access `mutiprocess = true` will print the error to `stderr` which is then displayed by `fzf`, in your case the inaccessible folders generate an error message which then causes an overflow with `fzf`.

The correct behavior should be to display the problematic folders and their corresponding error message at the top but this seems to bug fzf in your case (which may or may not be related to https://github.com/junegunn/fzf/issues/2697):

![screenshot-1641064546](https://user-images.githubusercontent.com/59988195/147858376-7867a18d-4550-4206-b6d1-961208b8da1a.png)

Can you test this with `skim`? If this works correctly with `sk` we will confirm this as an `fzf` bug.

**Comment by ibhagwan** [comment]
Solved it in #316, MacOS doesn't seem to like `stderr`, https://github.com/ibhagwan/fzf-lua/commit/e89d5bc009be8a631d5708701722fc30232879ee solves it.

**Comment by nyngwang** [comment]
Thank you!!!
